src/extractors/paths.py

```diff
@@ -0,0 +1,16 @@
+from algorithms.uast_to_bag_paths import Uast2BagOfPaths
+from sourced.ml.extractors import BagsExtractor, register_extractor
+from sourced.ml.algorithms import NoopTokenParser
+
+
+@register_extractor
+class BagOfPathsExtractor(BagsExtractor):
+    NAME = "id"
```

Comment:
change the name

Reply:
BagOfPathsExtractor? why? any suggestion?

Reply:
see comment above before, he meant the `NAME` variable I think which is already attributed to the identifier extractor and does not represent this one- although the class name should also be changed as it's vague. -> `NAME=code2vec` and `BagsOfPathsExtractor` -> `UastPathsBagExtractor`

Reply:
Yes, it is about `NAME` variable. `BagOfPathsExtractor` is fine.